lib/src/util/selectableColors.scss

```diff
@@ -6,53 +6,53 @@
  */
 .nx-selectable-color--light-blue, // deprecated
 .nx-selectable-color--sky {
-  --nx-selectable-color-dark: var(--nx-swatch-blue-80);
-  --nx-selectable-color-light: var(--nx-swatch-blue-90);
+  --nx-selectable-color-dark: var(--nx-color-selectable-color-sky-dark);
+  --nx-selectable-color-light: var(--nx-color-selectable-color-sky-light);
```

Comment:
Here too I think it would make sense to use swatches and `dark-mode` mixin calls directly rather than an additional layer of variables.

Reply:
> Here too I think it would make sense to use swatches and `dark-mode` mixin calls directly rather than an additional layer of variables.

2482cf1c45a68dee608e717600d57e33f811cd06